src/Compilers/CSharp/Portable/Symbols/ParameterSymbol.cs

```diff
@@ -385,6 +385,22 @@ internal sealed override ObsoleteAttributeData ObsoleteAttributeData
 
         internal abstract bool IsCallerMemberName { get; }
 
+        internal abstract AttributeAnnotations FlowAnalysisAnnotations { get; }
+
+        protected (bool memberHasExtra, AttributeAnnotations annotations) TryGetExtraAttributeAnnotations()
+        {
+            ParameterSymbol originalParameter = this.OriginalDefinition;
+            var containingMethod = originalParameter.ContainingSymbol as MethodSymbol;
+
+            if (containingMethod is null)
```

Comment:
Can we annotate parameters of properties?

Reply:
No, we currently only support extra annotations on methods.

---
In reply to: [187472420](https://github.com/dotnet/roslyn/pull/26656#discussion_r187472420) [](ancestors = 187472420)